src/CMakeLists.txt

```diff
@@ -745,6 +745,10 @@ list( APPEND SOURCES
       prefs/WaveformSettings.cpp
       prefs/WaveformSettings.h
 
+      # Sentry 
+
+      sentry/SentryHelper.h
```

Comment:
I'm not requiring a fix for this now:

This lonely header file will have to move to a library at some point, to enable the refactoring, so that everything dealing with the Sqlite DB can be separated from Audacity.

lib-sentry-reporting would be the obvious place for it, and then `${_OPT}has_sentry_reporting` would need testing in lib-sentry-reporting/CMakeLists.txt, and the library would build differently so that ADD_EXCEPTION_CONTEXT would be a no-op.  (It might even be a proper, inline function instead of a macro.)

Reply:
In the next commit, I will add `audacity_header_only_library` . It does not set any graph nodes yet.